src/components/Calendar.js

```diff
@@ -101,7 +101,7 @@ class Calendar extends PureComponent {
         <div className="rlc-days">
           {getDays(firstDayToDisplay).map(day =>
             <div
-              className={`rlc-day ${this.getClassNames(day)}`}
+              className={`rlc-day ${this.getClassNames(day)}${markedDays && markedDays.map(x => getDateWithoutTime(x)).includes(day) ? ' marked-day' : ''}`}
```

Comment:
All classnames are generated by `getClassNames` method, so you can manage `marked-day` on this method